openehr-rm/src/main/java/com/nedap/archie/rm/datavalues/quantity/DvQuantity.java

```diff
@@ -35,7 +36,7 @@
     @Deprecated
     @JsonIgnore
     @Nullable
-    private transient DvCodedText property;
+    private transient CodePhrase property;
```

Comment:
This has very little impact because nobody actually stores the property, but maybe postpone?